pkgs/applications/misc/twmn/default.nix

```
@@ -1,7 +1,8 @@
 { lib, mkDerivation, fetchFromGitHub, qtbase, qtx11extras, qmake, pkg-config, boost }:
 
 mkDerivation {
-  name = "twmn-git-2018-10-01";
+  pname = "twmn-git";
+  version = "2018-10-01";
```

```suggestion
  pname = "twmn";
  version = "git-2018-10-01";
```

---

is this really a good idea? i would call the package `twmn` or `twmn-git` and the version `2018-10-01` or `unstable-2018-10-01`. but the `git` part should ne be part of the version i think.

---

```suggestion
  pname = "twmn";
  version = "unstable-2018-10-01";
```
The `-git` is from [ages ago](https://github.com/NixOS/nixpkgs/commit/6cd330b3e7034ad99d9d0930249362677d7f7009), [current practice](https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming) is `unstable` in the name ([location is debatable](https://github.com/NixOS/nixpkgs/issues/68518) but `version = "unstable-..."` is the most widespread way rn).